src/main.rs

```diff
@@ -5,23 +5,15 @@
 
 use parsec_tool::err;
 
-use anyhow::Context;
-use anyhow::Result;
 use parsec_tool::cli;
 use structopt::StructOpt;
 
-fn run() -> Result<()> {
```

Comment:
Using `anyhow` after updating the client caused a bug, reason was that the updated `Error` type from the client does not implement some traits, including `Sync`.